helm/vernemq/values.yaml

```diff
@@ -231,12 +233,6 @@ additionalEnv:
 #    value: "10000"
 #  - name: DOCKER_VERNEMQ_MAX_OFFLINE_MESSAGES
 #    value: "-1"
-#  - name: DOCKER_VERNEMQ_LISTENER__SSL__CAFILE
```

Comment:
@dereulenspiegel can you please keep this comment in, so that we have "the old ways" still documented.

Reply:
@ioolkos Of course. But maybe it makes sense add a little comment warning of potential conflicts with the `certificate` config key?

Reply:
jep, sure, feel free to add that in place there. Thanks!

Reply:
I readded the removed example env config and added a little comment.